devsoc-unsw/jobsboard
# feat(frontend): revamp admin list pending jobs page [JOB-24]

## What it does
Users with administrator privileges are now able to:
- Preview a list of the pending jobs made by companies
- Preview the details of a specific job by clicking on the job card which shows a modal
- Approve individual pending jobs
- Reject individual pending jobs

## How to test
- Navigate to '/login/admin'
- Provide a valid username/password
- Navigate to 'Verify Job Post' (/admin/jobs/pending)
- Preview/approve/reject all pending jobs
